project/data_analysis/python/montecarlo/mc_get_spectra.py

```diff
@@ -192,20 +195,21 @@
                                                             spectra=spectra,
                                                             binned_mcm=binned_mcm)
 
-                            lb, ps = kspace.deconvolve_kspace_filter_matrix(lb,
-                                                                            ps,
-                                                                            kspace_transfer_matrix[f"{sv1}_{ar1}x{sv2}_{ar2}"],
-                                                                            spectra)
+                            if d["apply_kspace_filter"]:
```

Comment:
And here use the previous variable `apply_kspace_filter` otherwise you would apply the deconvolution for `HEALPIX` pixellisation